### DISCUSSION TITLE
Action not updated in PR

### DISCUSSION THREAD

**Discussion by unional** [community/community#40478] [Actions]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/3254987/204886185-3a94669a-e9bf-4cc0-ba5d-f885ae596d0b.png)
![image](https://user-images.githubusercontent.com/3254987/204886386-3d2777f3-9fb5-4bca-bb29-86f7a03ebce6.png)
![image](https://user-images.githubusercontent.com/3254987/204886257-809de6a6-9b7f-4f53-90f4-45ff167723e4.png)

https://github.com/unional/color-map/pull/148
https://github.com/unional/color-map/actions/runs/3568345406

You can see from the images and links above that the action has been executed, but the PR status are not updated.

Have not seen this before, might worth looking into.

**Comment by ossanna16** [comment]
Hi there @unional and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
